addons/hr_timesheet/report/timesheets_analysis_report.py

```diff
@@ -25,6 +25,14 @@ class TimesheetsAnalysisReport(models.Model):
     unit_amount = fields.Float("Time Spent", readonly=True)
     partner_id = fields.Many2one('res.partner', string="Partner", readonly=True)
     milestone_id = fields.Many2one('project.milestone', related='task_id.milestone_id')
+    message_partner_ids = fields.Many2many('res.partner', compute='_compute_message_partner_ids',
+                                           search='_search_message_partner_ids', readonly=True)
+
+    def _compute_message_partner_ids(self):
+        return self.env['account.analytic.line']._compute_message_partner_ids()
+
+    def _search_message_partner_ids(self, operator, value):
+        return self.env['account.analytic.line']._search_message_partner_ids(operator, value)
```

Comment:
Yep, you can indeed do that since the field is also in that report (I am not wrong). 🙂